README.md

```diff
@@ -1,10 +1,13 @@
-## Docker for Visual Studio Code
 
-[![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/Nightly/vscode-docker-nightly-2?branchName=master)](https://dev.azure.com/ms-azuretools/AzCode/_build/latest?definitionId=22&branchName=master)
+The Docker extension makes it easy to build, manage, and deploy containerized applications from Visual Studio Code. It also provides one-click debugging for Node.js, Python, and .NET Core.
+
+**Check out the [Working with containers](https://aka.ms/AA7arez) topic on the Visual Studio Code documentation site to get started**.
 
-The Docker extension makes it easy to build, manage and deploy containerized applications from Visual Studio Code.
+![Docker extension overview](resources/readme/overview.gif)
 
-**Check out [Working with containers](https://aka.ms/AA7arez) topic on Visual Studio Code documentation site to get started**.
+## Extension info
+
+[![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/Nightly/vscode-docker-nightly-2?branchName=master)](https://dev.azure.com/ms-azuretools/AzCode/_build/latest?definitionId=22&branchName=master)
```

Comment:
These badges should stay at the top. Every other project I've ever seen using badges has them at the top (or immediately under a title). Examples:

- https://github.com/microsoft/vscode-azurefunctions/blob/master/README.md
- https://github.com/OmniSharp/omnisharp-vscode/blob/master/README.md
- https://github.com/microsoft/vscode-eslint/blob/master/README.md

Reply:
I would actually put them at the very bottom of the doc. Only people who contribute code to the extension care about these badges, which is a very small percentage of our users. They take valuable real estate for no good reason.

Reply:
> These badges should stay at the top. Every other project I've ever seen using badges has them at the top (or immediately under a title).

I agree with @bwateratmsft and feel they should be at the top for consistency purposes with other projects.

> Only people who contribute code to the extension care about these badges, which is a very small percentage of our users.

For myself, even if I am not looking to contribute to a project, I use the badges to get a quick overview of a project's popularity. I am more inclined to use something that has a million downloads compared to something that has a thousand downloads.

Reply:
@rcjsuen good point about popularity, thx for pointing that out

Reply:
The reason to move them down was to gain real estate when the ReadMe is shown in VS Code, but I see the point about the download counts. Will move them back up.